spec/GraphQLOverHTTP.md

```diff
@@ -738,10 +735,15 @@ Note: The GraphQL specification
 and refers to the situation wherein a _GraphQL field error_ occurs as a partial
 response; it still indicates successful execution.
 
-## Processing the response
+## Processing a response
+
+In a typical environment, the source of a response may be some intermediary
+server such as an API gateway, a proxy, a firewall, etc.
```

Comment:
This appears to imply that typically the response comes from an intermediary.  I would disagree.  I think we should remove "typical" and reword the sentence so it flows better. Rough draft:

```suggestion
The source of a response may be an intermediary server, such as an API gateway, proxy, or
firewall, in certain environments.
```